src/libraries/Common/src/System/Security/Cryptography/IImportExportShape.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Security.Cryptography
+{
+#if DESIGNTIMEINTERFACES
```

Comment:
What's the purpose of this ifdef? This file appears to only be consumed into the S.S.Cryptography.csproj, which sets this.

Reply:
It’ll also be consumed in Microsoft.Bcl.Cryptography as the project gets further along